static/app/views/alerts/rules/issue/details/ruleDetails.tsx

```diff
@@ -73,11 +79,19 @@ class AlertRuleDetails extends AsyncComponent<Props, State> {
     }
   }
 
+  onRequestSuccess = ({stateKey, data}) => {
+    if (stateKey === 'rule') {
+      this.setSnooze(data.snooze);
+      this.setSnoozeCreatedBy(data.snoozeCreatedBy);
+    }
+  };
```

Comment:
not sure i like pulling these properties out, might make more sense to mutate the rule object